src/config.cpp

```diff
@@ -4,6 +4,13 @@
 #include "fmt/format.h"
 #include "recomp_port.h"
 
+std::filesystem::path concat_if_not_empty(const std::filesystem::path& parent, const std::filesystem::path& child) {
+    if (!child.empty()) {
+        return parent / child;
+    }
+    return child;
```

Comment:
very minor, but I think this is just as readable.

```suggestion
    return child.empty() ? parent : parent / child;
```

Reply:
I'd consider this change if this were a longer function, but since this is all the function does I prefer the longer form since I do find it a little easier to read at a glance.